Ginger/Ginger/Reports/HTMLReportTemplatePage.xaml

```diff
@@ -111,6 +111,15 @@
                                         <RadioButton x:Name="htmlShowFirstIterationOnRadioBtn" Content="Yes" Padding="5,0,0,0" Checked="htmlShowFirstIterationOnRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
                                     </StackPanel>
                                 </StackPanel>
+                                <StackPanel x:Name="htmlExecutionCalculationLevel" Margin="10,10,0,0" Orientation="Horizontal" ToolTip="Show all iterations execution results on HTML Report e.g. Flow control or retry mechanism iterations">
+                                    <Label Content="Execution/Pass Rate Calculation Level:" Style="{StaticResource @LabelStyle}" FontWeight="Bold" FontSize="12"/>
+                                    <StackPanel x:Name="htmlExecutionCalculationLevelBtnsPnl" Margin="10,0,0,0" Grid.Row="0" Grid.Column="0" Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+                                        <RadioButton x:Name="htmlExecutionCalculationLevelActivityRadioBtn" Tag="Activity" Content="Activity" Padding="5,0,15,0" Checked="htmlExecutionCalculationLevelActivityRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
+                                        <RadioButton x:Name="htmlExecutionCalculationLevelActionRadioBtn" Tag="Action" Content="Action" Padding="5,0,0,0" Checked="htmlExecutionCalculationLevelActionRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
+                                    </StackPanel>
+                                </StackPanel>
+                                <Label Style="{StaticResource @LabelStyle}" FontSize="10" Content="Note: Supported only in case of DB execution logger type is selected." Padding="15,0,0,0"/>
```

Comment:
change label content to be 
"Note: Supported only in case the selected Execution Logger type is DB."

as part of PR- change the "LiteDB" radio btn value in logger configurations page to "DB"